packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -40,6 +52,19 @@ initializeFluentUiIcons();
  */
 export type ContainerSummaryViewProps = HasContainerId;
 
+const columnsDef: TableColumnDefinition<Item>[] = [
+	createTableColumn<Item>({
+		columnId: "containerProperty",
+	}),
+	createTableColumn<Item>({
+		columnId: "value",
+	}),
+];
+interface Item {
+	property: string;
+	value: string;
+}
+const items = [];
```

Comment:
We'll want to move this state into the component(s) that use it. Otherwise, it won't be correctly cleaned up when usages are cleaned up.